Cargo.toml

```diff
@@ -14,13 +14,15 @@ version = "0.1.1"
 authors = ["paul@colomiets.name"]
 
 [dependencies]
-tokio-core = "0.1.10"
+tokio = "0.1.5"
 tokio-io = "0.1.3"
+tokio-timer = "0.2.1"
```

Comment:
Looks like unneeded dependency?

Reply:
It seems it really is. Have removed in additional commit.